File: app/src/main/java/com/xlzhen/sharefileserver/Application.java
package com.xlzhen.sharefileserver;

import android.content.Context;

/* loaded from: classes3.dex */
public class Application extends android.app.Application {
    public static Application application = null;
    public static String registerShareFile = null;
    public static String[] shareFiles = null;
    public static String clipDataNote = "";//复制文本Note

    public static void setClipDataNote(String str) {
        clipDataNote = str;
    }

    public static String getRegisterShareFile() {
        return registerShareFile;
    }

    public static String[] getShareFiles() {
        return shareFiles;
    }

    public static String getClipDataNote() {
        return clipDataNote;
    }

    @Override // android.app.Application
    public void onCreate() {
        super.onCreate();
        application = this;
    }

    public static Context getContext() {
        return application.getApplicationContext();
    }

    public static void registerShareFile(boolean register, String str) {
        registerShareFile = str;
    }

    public static void setShareFiles(String[] strArr) {
        shareFiles = strArr;
    }
}

File: app/src/main/java/com/xlzhen/sharefileserver/Application.kt
package com.xlzhen.sharefileserver

import android.content.Context

class Application : android.app.Application() {
    companion object {
        var application: Application? = null
        var registerShareFile: String? = null
        var shareFiles: Array<String>? = null
        var clipDataNote: String = "" // 复制文本Note

        fun getContext(): Context {
            return application!!.applicationContext
        }

        fun registerShareFile(register: Boolean, str: String?) {
            registerShareFile = str
        }
    }

    override fun onCreate() {
        super.onCreate()
        application = this
    }
}
